activerecord-spanner-adapter.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.5"
 
   spec.add_dependency "google-cloud-spanner", "~> 2.10"
-  spec.add_runtime_dependency "activerecord", "~> 6.1.4"
+  spec.add_runtime_dependency "activerecord", "~> 7.0.0"
```

Comment:
I've dropped support for ActiveRecord 6.1.x, but if you wish I think we can keep compatibility both with ActiveRecord 6.1.x and 7.0.x. If you prefer to do so, please let me know.

Reply:
I think we would like to keep compatibility both with ActiveRecord 6.1.x and 7.0.x. @olavloite What do you think?

Reply:
Ideally we'd want to keep compatibility with the last two major versions. If there are breaking changes between them, it should still be possible via logic in the code that branches based on the ActiveRecord version.